common/scripted_effects/00_scripted_effects.txt

```diff
@@ -5230,6 +5230,11 @@ remove_symptoms_effect = {
 		limit = { trait = malaise}
```

Comment:
> Removed healing from Twilight's Hammer.
The society says otherwise.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79054184-e7203980-7c53-11ea-82bb-e20d4455514d.png)

</details>